tools/create_release.py

```diff
@@ -151,6 +152,14 @@ def upload(self, path: Path, mimetype: str):
         response = requests.post(self.upload_url, headers=headers, params=params, data=path.read_bytes())
         response.raise_for_status()
 
+    def create_source_fallback(self):
+        response = requests.get(self.wrap_section['source_url'])
```

Comment:
Shouldn't we use meson subprojects download for this, which performs its own safety checks?

Reply:
Meson is not installed in release CI job and the url/checksum should already have been checked by the CI before merging. I think that should be good enough, no?

Reply:
Hmm, fair point.